documentation/modules/ROOT/pages/transformations/event-flattening.adoc

```diff
@@ -303,33 +303,22 @@ Optionally, you can override the field name via `<field name>:<new field name>`,
 When the SMT adds metadata fields to the simplified record's header, it prefixes each metadata field name with a double underscore. For a struct specification, the SMT also inserts an underscore between the struct name and the field name. +
  +
 If you specify a field that is not in the change event record, the SMT does not add the field to the header.
-|===
-
-The following table describes the options that you can specify to configure the event flattening SMT to integrate with the event changes SMT.
-
-
-.Descriptions of event flattening SMT configuration options for integration with event changes SMT
-[cols="30%a,25%a,45%a",subs="+attributes",options="header"]
-|===
-|Option
-|Default
-|Description
 
-|[[extract-new-record-state-header-changed-name]]xref:{link-event-flattening}#extract-new-record-state-header-unchanged-name[`header.unchanged.name`]
+[id="extract-new-record-state-drop-fields-header-name"]
+|xref:{link-event-flattening}#extract-new-record-state-drop-fields-header-name[`drop.fields.header.name`]
 |
-|The Kafka message header to source unchanged field names from.
+|The Kafka message header name to source fields that should be dropped.
 
-[id="extract-new-record-state-drop-unchanged-fields"]
-|xref:{link-event-flattening}#extract-new-record-state-drop-unchanged-fields[`drop.unchanged.fields`]
+[id="extract-new-record-state-drop-fields-from-key"]
+|xref:{link-event-flattening}#extract-new-record-state-drop-fields-from-key[`drop.fields.from.key`]
 |`false`
-|{prodname} generates a flattened event structure based on the `after` portion of the event's payload that includes all fields by default.
-For an `UPDATE` operation, this behavior can be adjusted to only include the fields that were changed by setting this option to `true` to drop or omit fields that were not changed in the update.
+|Controls whether fields are also be dropped fom the event's key.
```

Comment:
```suggestion
|Specifies whether you want the SMT to remove fields that are listed in `drop.fields.header.name` from the event's key.
```
I'm assuming that the fields to remove from the key are those that you specify in `drop.fields.header.name`?

Reply:
Correct.